cmd/neofs-adm/internal/modules/morph/subnet.go

```diff
@@ -105,18 +101,19 @@ func initSubnetClientCheckNotary(c *morphsubnet.Client, key *keys.PrivateKey, ch
 		return errors.New("missing endpoint")
 	}
 
-	// read contract address
-	contractAddr, err := util.Uint160DecodeStringLE(viper.GetString(flagSubnetContract))
-	if err != nil {
-		return fmt.Errorf("subnet contract address: %w", err)
-	}
-
 	// create base morph client
 	cMorph, err := client.New(key, endpoint)
 	if err != nil {
 		return err
 	}
 
+	const nnsSubnetKey = "subnet.neofs"
```

Comment:
`client.NNSSubnetworkContractName `?

Reply:
forget that it is a public const

reused it